packages/create-react-app/index.js

```diff
@@ -176,7 +198,13 @@ function run(root, appName, version, verbose, originalDirectory) {
       process.exit(1);
     }
 
-    checkNodeVersion(packageName);
+    var packageJsonPath = path.resolve(
+      process.cwd(),
+      'node_modules',
+      packageName,
+      'package.json'
+    );
+    checkNodeVersion(packageJsonPath);
```

Comment:
This lets us increase minimal version with time if `react-scripts` imposes this but user doesn't update the CLI itself.